
float32 bus_voltage
float32 cpu_temp
float32 external_temp
